packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -353,6 +361,9 @@ class FileNode extends TreeNode<FileNode> {
 }
 
 class ScriptRefUtils {
+  static String fileName(ScriptRef scriptRef) =>
+      Uri.parse(scriptRef.uri).path.split('/').last;
```

Comment:
would it be more performant if we cached this value on the ScriptRef rather than computing this each time?

Reply:
It's cached where it's used in the `FileNode`. This is simply a helper method on top of it.